client/signup/jetpack-connect/site-url-input.jsx

```diff
@@ -1,12 +1,17 @@
-import React from 'react';
+/**
+ * External dependencies
+ */
+ import React from 'react';
```

Comment:
good catch @gurdiga ! thank you a lot!

I'll fix it in a different PR : https://github.com/Automattic/wp-calypso/pull/4133